src/alto-ui/Datagrid/Datagrid.js

```diff
@@ -45,7 +45,16 @@ const FORMATTERS = {
           .toFormat('dd LLL yyyy')
       : '',
   number: (x, col, row, { locale }) =>
-    formatNumber(x, locale, col.precision || 0, null, col.disableThousandSeparator),
+    formatNumber(
+      col.percent ? x * 100 : x,
```

Comment:
@renaudtertrais i cannot understand this part why i cannot do this in formatNumber? if i manipulate this number in this method i get wrong results when i multiply number here its correct :)